compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+''''exec "python3" "$0" "$@" #'''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+
+
+def main():
+    work_dir = os.getcwd()
+    response = requests.get("https://api.github.com/repos/Samsung/ONE/tags")
+    if  response.status_code != 200 :
+        print("Error was occured while downloading ONE release tags")
+        print("status_code: ", response.status_code)
+        print("Error_reason: " + response.reason)
+        exit(1)
+    
+    versions_str = response.content
+    versions_json = json.loads(versions_str)
+    recent_version = versions_json[0]["name"]
```

Comment:
github api always returns latest version of release at the first of the content?

How about use https://docs.github.com/en/rest/releases/releases#get-the-latest-release?

It means,
```python
TAG_URL = "https://api.github.com/repos/Samsung/ONE/releases/latest" # NOTE HERE
try:
    response = requests.get(TAG_URL)
except requests.exceptions.RequestException as e:
    raise SystemExit(e)
```

Reply:
@holawan Even if we add the features about selecting version, I think we would use two URL later - latest and tags(or releases). Because we can't assume tags or releases api returns versions in the specific order.  

I mean that we should use `latest` URL in the first place.

Reply:
@mhs4670go 
Thank you for the detailed explanation. I'll check again tomorrow!